modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/users/provider/RoleUsersProvider.java

```diff
@@ -59,6 +63,18 @@ public List<User> provide(NotificationContext notificationContext)
 				notificationRecipientSetting.getCompanyId(),
 				notificationRecipientSetting.getValue());
 
+			if ((role.getType() == RoleConstants.TYPE_ORGANIZATION) ||
+				(role.getType() == RoleConstants.TYPE_SITE)) {
+
+				userIds.addAll(
+					ListUtil.toList(
+						_userGroupRoleLocalService.getUserGroupRolesByGroup(
+							notificationContext.getGroupId()),
+						UserGroupRoleModel::getUserId));
```

Comment:
@seltong could you please check if we can use the `_userLocalService.getGroupUserIds`?

Reply:
We can use the `_userLocalService.getGroupUserIds` to get the userIds that are linked to site roles, but we cannot use it to get userIds linked to organization roles.

I tryied to use the `_userLocalService.getOrganizationUserIds()`, but in this case we need a organizationId.

I'm inclined to go with the approach current approach.